projects/packages/my-jetpack/_inc/components/help-card/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { ExternalLink } from '@wordpress/components';
+import clsx from 'clsx';
+import styles from './styles.module.scss';
+
+export type HelpCardProps = {
+	title: string;
+	icon: React.ReactNode;
+	headingTag?: `h${ 1 | 2 | 3 | 4 | 5 | 6 }`;
```

Comment:
I see that this prop is not used anywhere just the default, will we ever need it or YAGNI?

Reply:
Yeah I get the idea, we can leave it, just sometimes future-looking additions are never used in the future